scene/animation/animation_player.cpp

```diff
@@ -199,7 +199,7 @@ void AnimationPlayer::_validate_property(PropertyInfo &p_property) const {
 void AnimationPlayer::_get_property_list(List<PropertyInfo> *p_list) const {
 	List<PropertyInfo> anim_names;
 
-	anim_names.push_back(PropertyInfo(Variant::DICTIONARY, "libraries"));
+	anim_names.push_back(PropertyInfo(Variant::DICTIONARY, PNAME("libraries")));
```

Comment:
Are you sure this one can be translated? Isn't it used programmatically by name? See also the next few lines with `next/` and `blend_times`.

Reply:
It's shown in the inspector. The other two have `PROPERTY_USAGE_NO_EDITOR` defined.